js/controller.js

```diff
@@ -192,6 +192,18 @@ OO.plugin("Html5Skin", function (OO, _, $, W) {
 
     onAuthorizationFetched: function(event, authorization) {
       this.state.authorization = authorization;
+
+      this.mb.subscribe(OO.EVENTS.VC_PLAYED, 'customerUi', _.bind(this.onVcPlayed, this));
```

Comment:
These events are subscribed during init method. We should avoid double subscription.